go/teams/rename.go

```diff
@@ -75,12 +75,13 @@ func RenameSubteam(ctx context.Context, g *libkb.GlobalContext, prevName keybase
 
 		g.Log.CDebugf(ctx, "RenameSubteam make sigs")
 		renameSubteamSig, err := generateRenameSubteamSigForParentChain(
-			g, me, deviceSigningKey, parent.chain(), subteam.ID, newName, admin)
+			libkb.NewMetaContext(ctx, g), g, me, deviceSigningKey, parent.chain(), subteam.ID, newName, admin)
```

Comment:
you should only need to call `libkb.NewMetaContext` once here.  and also, there's no need to also pass in `g` if you are passing in a `libkb.MetaContext`.